samples/FunctionApp/Program.cs

```diff
@@ -19,14 +16,8 @@ static async Task Main(string[] args)
 // #endif
             //<docsnippet_startup>
             var host = new HostBuilder()
-                //<docsnippet_configure_app>
-                .ConfigureAppConfiguration(c =>
-                {
-                    c.AddCommandLine(args);
-                })
-                //</docsnippet_configure_app>
                 //<docsnippet_middleware>
-                .ConfigureFunctionsWorkerDefaults()
+                .ConfigureFunctionsWorkerDefaults(args)
```

Comment:
@fabiocav -- wanted to make sure we were all cool with this before i update doc comments...

Reply:
Had a discussion offline